src/app/rundown-view/components/unfocused/unfocused.component.html

```diff
@@ -0,0 +1 @@
+<span class="label"> {{ outOfFocusMessage.toUpperCase() }}</span>
```

Comment:
Instead of localizing the string in the class and uppercase it here, you can localize it here and use CSS to uppercase, such that: this becomes:
```html
<span class="label" i18n>global.focus.out-of-focus</span>
```
And the SCSS file has:
```scss
&.unfocused {
  ...
  text-transform: uppercase;
  ...
}

Reply:
I changed it to follow the suggestion, however, i added it on the `label`-class instead!